packages/shared/providers.yaml

```diff
@@ -692,6 +692,12 @@ stripe:
     auth_mode: OAUTH2
     authorization_url: https://connect.stripe.com/oauth/authorize
     token_url: https://connect.stripe.com/oauth/token
+stripe-express:
```

Comment:
Does this require a new provider? It seems identical to the above entry for `stripe` and you could just add the `token_response_metadata` to that existing provider, no?

Reply:
The reason I created the PR was because Stripe has 2 different endpoints for their OAuth verification. There's the existing one `https://connect.stripe.com/oauth/authorize`, but this new one adds the "/express" subroute, becoming `https://connect.stripe.com/express/oauth/authorize` and leading to a different flow.

If there's another way to represent this in the providers list, I can definitely make the change.